tests/UnifiedSpecTests/Context.php

```diff
@@ -232,6 +236,20 @@ public function stopEventCollectors(): void
         }
     }
 
+    public function registerFailPoint(stdClass $failPoint, Server $server): void
+    {
+        $this->failPoints[] = [
+            'failPoint' => $failPoint,
+            'server' => $server,
+        ];
```

Comment:
Do you want to adopt Laravel' Style?
```suggestion
        $this->failPoints[] = compact('failPoint', 'server');
```

Reply:
This is Laravel style? I was using `compact()` back when Taylor was still a C# developer.

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExa2JsdHFocmZrbzNndmg4aTYwbW9tc2NkZ25mcXAzbGlrOWQ2ejd3dyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/P4sjB6SI2fErK/giphy.gif)

Reply:
This is Laravel style? I was using `compact()` back when Taylor was still writing C#.

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExa2JsdHFocmZrbzNndmg4aTYwbW9tc2NkZ25mcXAzbGlrOWQ2ejd3dyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/P4sjB6SI2fErK/giphy.gif)

----

On a serious note, I've considered using this in PHPLIB but opted to avoid the function call. It'd be less a concern in `tests/` than `src/`, though. I also wonder if the tooling is smart enough to not complain about variables being assigned and never referenced.

Reply:
> This is Laravel style?

Yep https://github.com/search?q=repo%3Alaravel%2Fframework%20compact&type=code

That was more a joke. I find it hard to read.

Reply:
Glad I waited with my response, that way I can keep my true feelings about `compact` to myself 🤫